### ISSUE TITLE
Accessibility certification guidelines for how to handle within publication files

### ISSUE THREAD

**Issue by RachelComerford** [w3c/publ-cg#62]
It has been proposed that there will be accessibility certification that would be generated by third parties and be allowed to be associated with files. We should discuss what the best practice might be around this and determine if standards/guidelines make sense for whether this be included in metadata within the file and/or human readable somewhere standard so that readers can be aware. 

**Marketplace Impact**
For all marketplaces that support accessible content

**Possible tech impact**
May need specific metadata elements and may need to be coordinated with other metadata standards orgs for dissemination of information that needs to be used for the marketplace and not just the content package

**Comment by deborahgu** [comment]
This should work in concert with the schema.org accessibility metadata -- no sense in further forking this effort -- but should also confront the feedback that the schema.org terms are difficult to understand and involve concepts which are complex and hard to understand.

**Comment by mattgarrish** [comment]
There is metadata already for this: http://www.idpf.org/epub/a11y/accessibility.html#sec-conf-reporting

It's not in schema.org as it wasn't realistic on the 3.1 timeline and there are issues about whether this should be solved more generally. There was work done on credentials in schema.org but was only a proposal, for example.

This should be solved within a revision of the accessibility spec, though, otherwise we risk forking practices.

**Comment by deborahgu** [comment]
Oh, thanks, Matt! I thought that was *in* the schema.org ontology; my
bad.
